packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +926,47 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting
+   * @param joinMeetingParams
+   */
+  export function joinMeeting(joinMeetingParams: JoinMeetingParams): Promise<string | object> {
+    return new Promise<string | object>((resolve) => {
+      ensureInitialized(runtime, FrameContexts.content);
+      resolve(
+        sendAndHandleStatusAndReasonWithVersion(
+          getApiVersionTag(ApiVersionNumber.V_2, ApiName.Meeting_JoinMeeting),
+          'meeting.joinMeeting',
+          joinMeetingParams,
+        ),
+      );
+    });
+  }
+
+  /** Join meeting parameters */
+  export interface JoinMeetingParams {
+    /** The meeting url to join. */
+    skypeTeamsMeetingUrl: string;
```

Comment:
`skypeTeamsMeetingUrl` should be a URL type to guarantee that developers are passing in a valid URL. You'll then have to take that URL and serialize it into a string before passing it to `sendAndHandleStatusAndReasonWithVersion`. For an example, see `marketplace.Item.imageURL` and the corresponding `serializeCartItems` function in `marketplaceUtils.ts`

Reply:
Changed the type to URL and accommodated it accordingly.